protocol/xmrmaker/instance.go

```diff
@@ -127,21 +129,9 @@ func (inst *Instance) abortOngoingSwap(s *swap.Info) error {
 func (inst *Instance) createOngoingSwap(s *swap.Info) error {
 	// check if we have shared secret key in db; if so, recover XMR from that
 	// otherwise, create new swap state from recovery info
-	sharedKey, err := inst.backend.RecoveryDB().GetSharedSwapPrivateKey(s.ID)
+	skA, err := inst.backend.RecoveryDB().GetCounterpartySwapPrivateKey(s.ID)
 	if err == nil {
-		kp, err := sharedKey.AsPrivateKeyPair() //nolint:govet
-		if err != nil {
-			return err
-		}
-		primaryCli := inst.backend.XMRClient()
-		destAddr := primaryCli.PrimaryAddress()
-		conf := primaryCli.CreateWalletConf("xmrmaker-swap-wallet-refund-ongoing")
-		err = sweepRefund(inst.backend.Ctx(), inst.backend.Env(), s.ID, conf, s.MoneroStartHeight, kp, destAddr)
-		if err != nil {
-			return err
-		}
-
-		return nil
+		return inst.completeSwap(s, skA)
```

Comment:
Does the swap info database entry need to get updated to completed after this?

Reply:
good point, I'll fix this. and yep I'll add the `recoveryDB.DeleteSwap()` call